### DISCUSSION TITLE
How to make an ad-hoc Node for Flowise

### DISCUSSION THREAD

**Discussion by aairom** [FlowiseAI/Flowise#3301] [Q&A]
Hi,
I want to develop an ad-hoc node.
I didn't find ant tutorials!
Thanks

**Comment by steve7158** [comment]
Hi I think this might be answered previously, but just incase you are still wondering or anyone else stumbles here, check this tutorial out: https://github.com/FlowiseAI/FlowiseDocs/blob/main/contributing/building-node.md
Cheers !
